libraries/botbuilder-dialogs-adaptive/src/sequenceContext.ts

```diff
@@ -27,10 +27,10 @@ export interface ActionChangeList {
 }
 
 export enum ActionChangeType {
-    insertActions = 'insertActions',
-    appendActions = 'appendActions',
-    endSequence = 'endSequence',
-    replaceSequence = 'replaceSequence'
+    insertActions = 'InsertActions',
+    appendActions = 'AppendActions',
+    endSequence = 'EndSequence',
+    replaceSequence = 'ReplaceSequence'
```

Comment:
This is actually a bug in C#. We should be consistent with regards to case and use cammelCase here.

Reply:
Just spoke with Tom & Carlos and they agree. They're going to fix the serialization logic on the C# side of things to ensure that all enums are cammelCase. 

---
In reply to: [363432824](https://github.com/microsoft/botbuilder-js/pull/1483#discussion_r363432824) [](ancestors = 363432824)

Reply:
> Just spoke with Tom & Carlos and they agree. They're going to fix the serialization logic on the C# side of things to ensure that all enums are cammelCase.
> 
> In reply to: [363432824](https://github.com/microsoft/botbuilder-js/pull/1483#discussion_r363432824) [](ancestors = 363432824)

We would use cammelCase in .dialog files so that it can be parsed in both dotnet sdk and js sdk, right? If so, I would rename them to cammelCase.